src/js/main.js

```diff
@@ -150,7 +150,8 @@ var main = main || {};
                 method: 'POST',
                 contentType: 'text/plain',
                 data: data
-            }).done(function(response) {                
+            }).done(function(response) {
+                response = ('<p>Source Document: <a href="' + item.html_url + '"target=”_blank”>' + item.html_url + '</a></p>'.concat(response));
```

Comment:
Put part of this in the HTML.
```
<p>Source Document: <a id="whatever-id-you-come-up-with" target="_blank"></a></p>
```
Then use jquery to get the anchor element and update it. Don't tweak the response, just pass it straight.

Reply:
fixed